### DISCUSSION TITLE
アプリアイコンのアイデア

### DISCUSSION THREAD

**Discussion by KoukiNAGATA** [jphacks/B_2121_client#44] [Ideas]
#2 だと閉じるタイミングがわかりづらいのでDiscussionsに置いてみた
アプリのprimaryカラーなど、アイコン意外に関するアイデアもあればおねがいします！
↓アイコン案1
![penguin](https://user-images.githubusercontent.com/38304075/139016332-8055477a-2651-4ae8-99a3-972fb523ee93.png)

**Comment by KoukiNAGATA** [comment]
もうちょっと丸くしたい

**Comment by Eri-0910** [comment]
ご飯感はあるけど、コミュニティ感はないよなぁと思ったり

**Comment by KoukiNAGATA** [comment]
コミュニティ感、例えばペンギンを2匹に増やすとアイコンキツキツになるのが苦しい
むずかしい...
